addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -202,6 +203,65 @@ const PopupWidget = publicWidget.Widget.extend({
 
 publicWidget.registry.popup = PopupWidget;
 
+publicWidget.registry.cookies_bar = PopupWidget.extend({
+    selector: '#website_cookies_bar',
+    start() {
+        if (
+            this.el.querySelector(".o_cookies_popup")
+            && window.location.pathname === new URL(
+                this.el.querySelector(".o_cookies_bar_text_policy").href
+            ).pathname
+        ) {
+            // Stable fix: using raw HTML instead of a template.
+            this.toggleEl = $(`
```

Comment:
Maybe there is no need for a template or jQuery for this :thinking: Just create an element in JS, add `className`, apply `this._toggleCookiesBar()`…?

Reply:
I think I started like this, it felt less practical (± 10 lines just for that, less readable). But actually that might be better. :thinking:

Reply:
On 2nd thought, I'm not sure. Using a template in master (and jQuery before to have a closer match between versions) would probably be less painful when the time comes to migrate snippets to OWL.
What do you think?